Objects/typeobject.c

```diff
@@ -3634,8 +3634,9 @@ check_basicsize_includes_size_and_offsets(PyTypeObject* type)
 }
 
 PyObject *
-PyType_FromMetaclass(PyTypeObject *metaclass, PyObject *module,
-                     PyType_Spec *spec, PyObject *bases_in)
+_PyType_FromMetaclass_impl(
```

Comment:
This function should be file local, so add the `static` keyword.

Also, file local helpers are normally lower-case only, and without a `Py` or `_Py` prefix.